src/vs/code/electron-main/app.ts

```diff
@@ -731,7 +731,7 @@ export class CodeApplication extends Disposable {
 
 	private async handleProtocolUrl(windowsMainService: IWindowsMainService, urlService: IURLService, uri: URI, options?: IOpenURLOptions): Promise<boolean> {
 		this.logService.trace('app#handleProtocolUrl():', uri.toString(true), options);
-
+		const accessibilitySupportEnabled: boolean = (<any>global).accessibilitySupportEnabled;
```

Comment:
Is the [current property](https://github.com/microsoft/vscode/blob/224472b8b914862214db6faa9385afa40427acf8/src/vs/platform/windows/electron-main/windowsMainService.ts#L1388) on the `INativeWindowConfiguration` not sufficient , why do we need to specify them at every open call site ?